pkg/asset/cluster/tfvar.go

```diff
@@ -0,0 +1,83 @@
+package cluster
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+	"github.com/openshift/installer/pkg/types/config"
+)
+
+const (
+	tfvarFilename  = "terraform.tfvar"
+	tfvarAssetName = "Terraform Variables"
```

Comment:
I [still][1] don't think we need a constant for this (folks who need it should just use the `Name()` method).  But we can adjust that later if we care.

[1]: https://github.com/openshift/installer/pull/273#discussion_r218936981